axolotl-web/src/main.js

```diff
@@ -8,6 +8,7 @@ import translations from '../translations/translations.json'
 import { library } from '@fortawesome/fontawesome-svg-core'
 import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
 import linkifyHTML from 'linkify-html'
+import "bootstrap";
```

Comment:
Nitpick: On the imports just above single quotes are used, here however double-quotes are used.
Switching to single quotes here would align all imports to use the same.

Reply:
This comment is valid also for the imports of the layouts in the router.

Reply:
Yes, you are right, i am also somehow confused by when to use which quotes. I have the feeling it's a cultural difference, like European languages/developers prefer the double-quotes and the American examples use the single quotes.